app/audit/AuditService.scala

```diff
@@ -129,6 +129,14 @@ class AuditService @Inject() (auditConnector: DefaultAuditConnector) {
       auditPayload   = baseAuditPayload(c, operator) + ("comment" -> "Liability case created")
     )
 
+  def auditCaseUpdated(originalCase: Case, updatedCase: Case, operator: Operator)(implicit hc: HeaderCarrier): Unit =
+    sendExplicitAuditEvent(
+      auditEventType = CaseUpdated,
+      auditPayload = baseAuditPayload(originalCase, operator) + (
+        "updatedCase" -> updatedCase.toString
```

Comment:
I think this should include the full `originalCase` and `updatedCase` JSON so that it's possible to see what has changed

Reply:
Thanks. Resolved